### PR TITLE
Task sf/j127 edit create episode page styles

### PR DESCRIPTION
**Pull request by Sayonarko**
edit-page

### CODE REVIEW

**Comment by Sayonarko** [comment]
[Edit/Create Episode Page styles](https://trello.com/c/zXKDao1m/127-edit-create-episode-page-styles)

File: frontend/src/components/common/select/select.tsx [hunk 1]
```diff
@@ -39,6 +40,7 @@ const Select: React.FC<Props> = ({ options, label, name, control, errors }) => {
         options={options}
         value={currentValue}
         onChange={handleSelectChange}
+        className={className}
```
Comment (what1s1ove) [inline frontend/src/components/common/select/select.tsx]: you don't use it ?🤔

File: frontend/src/components/configurate-episode/components/create-episode-form/create-episode-form.tsx [hunk 2]
```diff
@@ -8,98 +8,111 @@ import {
...
         <Input
           type={InputType.TEXT}
           label="Description"
-          placeholder="Enter episode description"
+          placeholder="Enter description episode"
           name={EpisodePayloadKey.DESCRIPTION}
           control={control}
           errors={errors}
+          hasMultipleRows
+          rows={7}
```
Comment (what1s1ove) [inline frontend/src/components/configurate-episode/components/create-episode-form/create-episode-form.tsx]:
```suggestion
```
jest left it as we have now 🙂

File: frontend/src/components/common/input/input.tsx [hunk 3]
```diff
@@ -18,6 +18,7 @@ type Props = {
   control: Control;
   errors: FieldErrors;
   hasMultipleRows?: boolean;
+  rows?: number;
```
Comment (what1s1ove) [inline frontend/src/components/common/input/input.tsx]:
```suggestion
```

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**